src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -11494,5 +11495,671 @@ .maxstack  0
   IL_0000:  ret
 }");
         }
+
+        internal static void CheckNames(IEnumerable<MetadataReader> readers, IEnumerable<StringHandle> handles, params string[] expectedNames)
+        {
+            var actualNames = readers.GetStrings(handles);
+            AssertEx.Equal(expectedNames, actualNames);
+        }
+
+        internal static void CheckNames(MetadataReader reader, IEnumerable<StringHandle> handles, params string[] expectedNames)
+        {
+            CheckNames(new[] { reader }, handles, expectedNames);
+        }
+
+
+        [Fact]
+        public void LambdaWithDefaultParameter()
+        {
+            var source = """
+using System;
+
+class Program
+{
+    // static void Report(object d) => Console.WriteLine(d.GetType());
```

Comment:
Dead code?

Reply:
Technically zombie code. It's dead yet still in the repo so sort-of alive therefore zombie